contents/data_vis_makie_cheatsheets.md

```diff
@@ -0,0 +1,19 @@
+## Cheat Sheets {#sec:makie_cheat_sheets}
```

Comment:
Can we change this file name to `makie_cheet_sheets` or something similar? Then the cheat sheets are available at 
<https://juliadatascience.io/makie_cheat_sheets> instead of 
<https://juliadatascience.io/data_vis_makie_cheatsheets>.

Reply:
yes, will do.